consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/validation/NewRoundMessageValidatorTest.java

```diff
@@ -227,20 +227,6 @@ public void lastestPreparedCertificateMatchesNewRoundProposalIsSuccessful() {
     final SignedData<ProposalPayload> proposal =
         proposerMessageFactory.createSignedProposalPayload(roundIdentifier, proposedBlock);
 
-    final ConsensusRoundIdentifier preparedRound =
```

Comment:
this needs to go back in, the preparedCertificate should be being used in the RoundChange

Reply:
Are you saying that the test 'should' be using the `preparedCert` but wasn't?

Reply:
...as the removed code, definitely was unreferenced.i.e. not being used

Reply:
After discussion: will rewrite the test to have both certs in the proposal